src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2062,6 +1995,28 @@ MavlinkReceiver::handle_message_rc_channels_override(mavlink_message_t *msg)
 	rc.values[16] = man.chan17_raw;
 	rc.values[17] = man.chan18_raw;
 
+#if defined(ATL_MANTIS_RC_INPUT_HACKS)
```

Comment:
Can't most of this be done with regular RC calibration parameters carried in the airframe parameters?

Reply:
I find it hard:
- Buttons are encoded as bits.
- Values are from 0..4095. Our calibration doesn't really support that, and if we change it to support it, then QGC still doesn't work with it because the values are out of range for it's detection as well as visualization.

Reply:
Fair enough, maybe we'll come up with something later.